browser/resources/settings/brave_tor_page/brave_tor_bridges_dialog.ts

```diff
@@ -67,20 +74,19 @@ class RequestBridgesDialog extends RequestBridgesDialogBase {
     this.captchaResolve_ = ''
     this.status_ = this.i18n('torRequestBridgeDialogWaiting')
     this.captcha_ = ''
-    this.enableSubmit_(false)
+
```

Comment:
Should we set `renewDisabled_` to `true` here so that the button isn't clicked twice?

Reply:
Yes, done